### DISCUSSION TITLE
clear cached value on update of setting

### DISCUSSION THREAD

**Discussion by inishantthakur** [tomatophp/filament-settings-hub#12] [Ideas]
I've got setting hub installed as part of ecommerce package, after some use and playing around I've found there is no convenient way to  invalidate or clear cache..

**Comment by tansautn** [comment]
It's feature of `spatie/laravel-settings`. The package that setting-hub wrapped on.
![image](https://github.com/user-attachments/assets/7ddb2016-fc2d-41d4-94fa-27924c09a3cc)

Did you tried these ?
